index.js

```diff
@@ -68,7 +68,39 @@ function toggleColor(element) {
   }
 }
 
+const header = document.querySelector("#header")
+
+header.addEventListener("click", function(){
+  toggleColor(header)
+})
+
 /***** Deliverable 2 *****/
 
+const button = document.querySelector(".like-button")
+const likes = document.querySelector(".likes")
+
+button.addEventListener("click", function(){
+  const likesCount = parseInt(likes.textContent)
+  likes.textContent = likesCount + 1 + " Likes"
```

Comment:
Looks good! The key here is that you have to read some information from the DOM to get the current number of likes, increment it, and then write it back to the DOM. You can also do some string interpolation here (just a matter of preference): 

```js
likes.textContent = `${totalLike + 1} Likes`
```